plugins/woocommerce/composer.json

```diff
@@ -49,7 +53,8 @@
 		"woocommerce/action-scheduler": "3.9.3",
 		"woocommerce/blueprint": "*",
 		"woocommerce/email-editor": "1.6.0",
-		"wordpress/abilities-api": "^0.1.1"
+		"wordpress/abilities-api": "^0.1.1",
+		"wordpress/mcp-adapter": "dev-trunk#7a2d22cff92328bc94f5b1648a66ae4273e949c5"
```

Comment:
Just noting that we have to use this because the current mcp-adapter package is only PHP 8+ compatible, while the trunk includes PHP 7.4 compatibility fixes.